KomodoPlatform/atomicDEX-Desktop
# tweak tooltip

Closes https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1289
![image](https://github.com/KomodoPlatform/atomicDEX-Desktop/assets/35845239/c1fdfb23-8a6f-41d6-b037-5870a8322018)

Its a minor improvement for clarity, but not the best visually. We can make it look pretty in a later version with some feedback from the design team.

To test:
- Launch
- Go to simple view
- hover over tradable balance
- see tooltip
- check full balance matches portfolio page
